content/en/docs/concepts/storage/volumes.md

```diff
@@ -121,14 +121,13 @@ If the EBS volume is partitioned, you can supply the optional field `partition:
 
 #### AWS EBS CSI migration
 
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
```

Comment:
I noticed that the "Complete" flag is still alpha. I guess we go straight to GA after 2 releases?

Reply:
The complete flag is deprecated. Right now we only have `InTreePluginAWSUnregister` flag. And I thought we already concluded that this flag will remain alpha forever and then removed it because we will delete these in-tree plugins code anyway. So no need to go GA for that flag